### ISSUE TITLE
This package..

### ISSUE THREAD

**Issue by ndelangen** [UsulPro/storybook-styleguidist#1]
My react when trying your demo:
😕 > 🤔 > 😯 > 😃

To explain:
1. Why would anyone want to do this? maybe when migrating away from storybook to styleguidist?
2. Wait a minute styleguidist will render everyone on one page, and that's what this thing is doing...
3. We want some way of switching to a "show me all stories"-mode
4. This could be a way of combining storybook with styleguidist.

@sapegin

**Comment by sapegin** [comment]
Yeah, I think that’s the right way to do it ;-) Can we automate it even further? Do we need to change anything in Styleguidist? Most likely loading Markdown examples from memory is not possible now.
